pym/portage/dbapi/bintree.py

```diff
@@ -672,17 +693,17 @@ def _populate(self, getbinpkgs=False, getbinpkg_refresh=True):
 					multi_instance = False
 					invalid_name = False
 					build_id = None
-					if myfile.endswith(".xpak"):
+					if myfile.endswith("." + self.suffix):
 						multi_instance = True
 						build_id = self._parse_build_id(myfile)
```

Comment:
Again, we really don't have to touch the suffix code in order to choose compression, so let's do that later in a separate patch.